man/liftoffrc.5

```diff
@@ -157,9 +157,9 @@ default:
 Specify the dependency managers to use.
 .Pp
 Currently, this accepts
-.Ic cocoapods
+.Ic cocoapods, carthage
```

Comment:
this should be:

```doc
.Ic cocoapods ,
.Ic carthage ,
and/or
.Ic bundler
```

`.Ic` is like putting the word that follows it in backticks.